inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java

```diff
@@ -372,24 +372,53 @@ public List<? extends StreamSink> listByCondition(SinkPageRequest request, UserI
     @Transactional(rollbackFor = Throwable.class)
     public Boolean update(SinkRequest request, String operator) {
         LOGGER.info("begin to update sink by id: {}", request);
-        this.checkParams(request);
-        Preconditions.checkNotNull(request.getId(), ErrorCodeEnum.ID_IS_EMPTY.getMessage());
-
-        // Check if it can be modified
-        String groupId = request.getInlongGroupId();
-        String streamId = request.getInlongStreamId();
-        String sinkName = request.getSinkName();
-        groupCheckService.checkGroupStatus(groupId, operator);
-
+        if (request == null) {
```

Comment:
For the "Service" layer method, it is recommended to put the parameter verification in the "Controller", because the external access is only the "Controller".

The calls of other "Service" layers in the system are all trusted data, and no more judgment is needed.

Reply:
Agree